install/kubernetes/tetragon/values.yaml

```
@@ -71,6 +71,8 @@ tetragon:
   # Tetragon puts processes in an LRU cache. The cache is used to find ancestors
   # for subsequently exec'ed processes.
   processCacheSize: 65536
+  # If you want to run Tetragon in debug mode change this value to true
```

```suggestion
  # -- If you want to run Tetragon in debug mode change this value to true
```
Non-blocking, but prefixing the comment with `-- ` will include it in generated docs (see also #2625).

---

RIght, I forgot about that. Thanks! Just `make -C install/kubernetes` and squashed the commits.